airbyte-webapp/src/config/defaultConfig.ts

```diff
@@ -27,7 +27,7 @@ const defaultConfig: Config = {
   segment: { enabled: true, token: "" },
   healthCheckInterval: 20000,
   version: "dev",
-  apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api/v1/`,
+  apiUrl: `${window.location.protocol}//${window.location.hostname}:8001/api`,
```

Comment:
Removing `/v1/` so we're no longer hard-coded to it.

Reply:
Trailing slash needed or not needed?

Reply:
Won't be needed at the moment: https://github.com/airbytehq/airbyte/blob/kc/generated-api-client/airbyte-webapp/src/core/request/apiOverride.ts#L49 And actually we should try to go without them for all changes, since most of the API urls should include it already (and if not that part of the code handles that).